src/browser/modules/Stream/SchemaFrame.jsx

```diff
@@ -79,49 +129,26 @@ export class SchemaFrame extends Component {
     }
   }
 
-  formatIndexAndConstraints (indexes, constraints) {
-    let indexString
-    let constraintsString
-
-    if (indexes.length === 0) {
-      indexString = 'No indexes'
-    } else {
-      indexString = 'Indexes'
-      indexString += indexes.reduce((acc, index) => {
-        acc += `\n  ${replace(
-          index.description,
-          'INDEX',
-          ''
-        )} ${index.state.toUpperCase()} ${
-          index.type === 'node_unique_property'
-            ? ' (for uniqueness constraint)'
-            : ''
-        }`
-        return acc
-      }, '')
-    }
-
-    if (constraints.length === 0) {
-      constraintsString = 'No constraints'
-    } else {
-      constraintsString = 'Constraints'
-      constraintsString += constraints.reduce((acc, constraint) => {
-        acc += `\n  ${constraint.description.replace('CONSTRAINT', '')}`
-        return acc
-      }, '')
-    }
+  render () {
+    const indexes = formatIndexes(this.state.indexes)
+    const constraints = formatConstraints(this.state.constraints)
 
-    return `${indexString}\n\n${constraintsString}\n`
-  }
+    const frame = (
+      <Slide>
+        <SchemaTable name='Indexes' content={indexes} />
+        <SchemaTable name='Constraints' content={constraints} />
+        <br />
+        <p className='lead'>Execute the following query for more information</p>
```

Comment:
The links doesn't really show _more_ information about the indexes and constraints, but rather semi-related info.
I would probably rewrite this to be something in the lines of: `Execute the following command to visualize what's related, and how` (we use that phrase in the sidebar "Saved scripts" -> "Data profiling").